MediaBrowser.Model/Plugins/PluginStatus.cs

```diff
@@ -0,0 +1,18 @@
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+#pragma warning disable SA1602 // Enumeration items should be documented
```

Comment:
These warnings should be fixed instead of disabled

Reply:
Removed and comments added.